Assets/SolarSystem/Planets/LODs/Planet_LODs.mat

```diff
@@ -336,7 +336,7 @@ Material:
       second: {r: 1, g: 0, b: 0, a: 0}
     - first:
         name: _SunPosition
-      second: {r: -0.019818218, g: -0.09801179, b: 1.495989, a: 0}
```

Comment:
This change doesn't seem to have anything to do with your PR and is probably a side effect of Unity's editor. There are pending changes in the Phone branch that will help with Unity's editor and material modification. Please remove this file from your PR.